fastlane/Fastfile

```diff
@@ -197,4 +197,49 @@ platform :ios do
     )
   end
 
+  desc "Build Perf-test app without Sentry"
+  lane :build_perf_test_app_plain do
+
+    setup_ci
+
+    sync_code_signing(
+      type: "appstore",
+      readonly: true,
+      app_identifier: ["io.sentry.cocoa.test-app-plain"]
+    )
+
+    build_app(
+      project: "Tests/Perf/test-app-plain/test-app-plain.xcodeproj",
+      scheme: "test-app-plain",
+      include_bitcode: false,
+      include_symbols: false,
+      export_method: "app-store",
```

Comment:
Likewise s/appstore/development, which is the default for `build_app.export_method`:
```suggestion
```